### PR TITLE
Update CCP info/links for Cloud Block Storage: INFODEV-2693

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by RackerKate** [comment]
@cyrichardson Another set for your review.

**Comment by cyrichardson** [comment]
@RackerKate Reviewing this one now.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b169410b1c/how-to/)

File: content/cloud-databases/connect-to-a-cloud-databases-instance.md [hunk 1]
```diff
@@ -185,14 +199,17 @@ Copy the following PHP script and paste it into a text editor:
...
 Databases instance. Do not add other nodes to the load balancer.
 
 #### Copy the instance hostname
 
-1.  Log in to the [Cloud Control Panel](http://mycloud.rackspace.com/).
-2.  In the top navigation bar, select **Databases &gt; Database
-    Instances**.
+Use the following steps to copy the instance hostname:
+
+1.  Log in to the [Cloud Control Panel](http://login.rackspace.com/).
```
Comment (catlook) [inline content/cloud-databases/connect-to-a-cloud-databases-instance.md]: missing a step...need to select rackspace cloud before selecting databases

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-3dd1d91403/how-to/)

**Pull request merged**